src/growthbook.ts

```diff
@@ -104,15 +104,7 @@ export class Feature<T extends JSONValue> {
 }
 
 const digestFeatureBaseConfig = {
-  templateId: 'd-328d1104d2e04fa1ab91e410e02751cb',
-  meta: {
-    from: {
-      email: 'informer@daily.dev',
-      name: 'daily.dev',
-    },
-    category: 'Digests',
-    asmGroupId: 23809,
-  },
+  templateId: '46',
```

Comment:
Don't we need to update more templateId's?
Or we'll do the main ones seperate later?